src/Modera/MjrIntegrationBundle/DependencyInjection/Configuration.php

```diff
@@ -122,6 +122,9 @@ public function getConfigTreeBuilder()
                 ->scalarNode('app_base_class')
                     ->defaultValue('')
                 ->end()
+                ->scalarNode('moment_js_version')
```

Comment:
Optionally, but probably I would add a comment that this feature has been added since 2.56.0 release - "// since 2.56.0"